proto/common_proto.piqi_pb.js

```diff
@@ -1038,6 +1047,8 @@ $root.common_proto = (function() {
                 writer.uint32(/* id 8, wireType 2 =*/66).string(message.messageTypeId);
             if (message.messageTsUs != null && message.hasOwnProperty("messageTsUs"))
                 writer.uint32(/* id 9, wireType 5 =*/77).fixed32(message.messageTsUs);
+            if (message.applicationId != null && message.hasOwnProperty("applicationId"))
```

Comment:
Should this be message.applicaitonId != null | "" ?

Reply:
this is an autogenerated code by `protobufjs`

Reply:
OIC. I'll trust in protobuf then 👍